ui/Screen/Settings.svelte

```diff
@@ -0,0 +1,144 @@
+<script>
+  import { Title, Text, Button } from "../DesignSystem/Primitive";
+  import { SidebarLayout, SegmentedControl } from "../DesignSystem/Component";
+
+  const themeOptions = [
+    {
+      title: "Light",
+      value: 0,
+    },
+    {
+      title: "Dark",
+      value: 1,
+    },
+  ];
+  const networkOptions = [
```

Comment:
If you want to consider options that we need for now it's:

* Emulator - for local in-memory
* FFnet - for the currently running one the team presented
* Testnet - coming soon(tm)

Reply:
:+1: